Cargo.toml

```diff
@@ -1,24 +1,7 @@
-[package]
-name = "stellar-liqpool-contract"
-version = "0.0.0"
-edition = "2021"
+[workspace]
+resolver = "2"
 
-[lib]
-crate-type = ["cdylib", "rlib"]
-
-[features]
-default = ["export"]
-export = []
-external = ["stellar-token-contract/external", "dep:ed25519-dalek", "dep:num-bigint", "dep:sha2", "dep:stellar-xdr"]
-
-[dependencies]
-ed25519-dalek = { version = "1.0.1", optional = true }
-num-bigint = { version = "0.4", optional = true }
-sha2 = { version = "0.10.2", optional = true }
-stellar-token-contract = { git = "https://github.com/stellar/rs-stellar-token-contract", rev = "03959d2", default-features = false  }
-stellar-contract-sdk = { git = "https://github.com/stellar/rs-stellar-contract-sdk", rev = "d279737" }
-stellar-xdr = { git = "https://github.com/stellar/rs-stellar-xdr", rev = "94e01c7b", features = ["next", "std"], optional = true }
-
-[dev_dependencies]
-rand = { version = "0.7.3" }
-stellar-liqpool-contract = { path = ".", default-features = false, features = ["external"] }
+members = [
+    "examples/liquidity_pool",
+    "examples/single_offer",
```

Comment:
This repo will become the examples repo. Since the only things in this repo should be examples, can we put the member projects in the root rather than in an examples folder? Where we used an examples folder in the sdk repo was only because there were also other things in the repo. It'll be more immediately satisfying if when a developer goes to the example repo they see a list of examples in front of them at the top level.

Reply:
Done